packages/ckeditor5-image/src/image/imageediting.js

```diff
@@ -110,27 +79,9 @@ export default class ImageEditing extends Plugin {
 						return value;
 					}
 				}
-			} )
-			.add( viewFigureToModel() );
+			} );
 
 		editor.commands.add( 'imageInsert', new ImageInsertCommand( editor ) );
```

Comment:
This command is inserting a block image or an inline image? Is it supposed to handle both types?

Reply:
I am handling this in another ticket about insert/upload images.